WordPress/src/main/java/org/wordpress/android/ui/domains/management/DomainStatusRow.kt

```diff
@@ -18,70 +18,107 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.graphicsLayer
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.tooling.preview.PreviewParameter
 import androidx.compose.ui.tooling.preview.PreviewParameterProvider
 import androidx.compose.ui.unit.dp
+import org.wordpress.android.fluxc.network.rest.wpcom.site.DomainStatus
+import org.wordpress.android.fluxc.network.rest.wpcom.site.StatusType
 import java.time.LocalDate
 import java.time.format.DateTimeFormatter
 import java.time.format.FormatStyle
 
 @Composable
 fun StatusRow(
-    status: DomainStatus,
-    expiry: LocalDate,
+    status: DomainStatus?,
+    expiry: LocalDate?,
     modifier: Modifier = Modifier,
 ) {
     Row(
         modifier = modifier,
         verticalAlignment = Alignment.CenterVertically,
     ) {
-        Box(
-            modifier = Modifier
-                .size(8.dp)
-                .clip(CircleShape)
-                .background(status.statusUiState.indicatorColor),
-        )
-        Spacer(modifier = Modifier.width(8.dp))
-        Text(
-            text = status.statusUiState.text,
-            color = status.statusUiState.textColor,
-            style = if (status.statusUiState.isBold) {
-                LocalTextStyle.current.copy(fontWeight = FontWeight.Bold)
-            } else {
-                LocalTextStyle.current
-            },
-        )
-        Text(
-            text = "Expires ${expiry.mediumFormat}",
-            textAlign = TextAlign.End,
-            color = MaterialTheme.colorScheme.secondary,
-            modifier = Modifier.weight(1f),
-        )
+        status?.also { status ->
+            BulletPoint(color = status.indicatorColor)
+            Spacer(modifier = Modifier.width(8.dp))
+            Text(
+                text = status.status ?: "Unknown",
```

Comment:
Should `"Unknown"` be also moved to `strings.xml`?

Reply:
Good catch! I left this here as a sort of "placeholder" until we figure out what to do about the status strings. The current plan is to have them translated on the backend, but in case the response is empty or null for this field. In this case, the string should be "Error", so this has been resolved w/ https://github.com/wordpress-mobile/WordPress-Android/pull/19269/commits/92930619e31ac0c64f05390ee98289bd29b06bae.